syntaxes/graphql.ex.json

```diff
@@ -35,6 +35,17 @@
           ]
         }
       ]
+    },
+    {
+      "name": "taggedTemplates",
+      "contentName": "meta.embedded.block.graphql",
+      "begin": "\\~(gql|graphql|GQL|GRAPHQL)(\"\"\")",
```

Comment:
I have added both high and low cases because each definition has different meanings. About adding `gql` and `graphql` I have seen both implementations in different code bases over time. The intention is to make everyone happy by default andavoidingd code rewrites if users want to try `vscode-graphql` extension.